tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from app import create_app
+from app import db
+from flask.signals import request_finished
+from app.models.planet import Planet
+
+@pytest.fixture
+def app():
+    app = create_app({"TESTING": True})
+
+    @request_finished.connect_via(app)
+    def expire_session(sender, response, **extra):
+        db.session.remove()
+
+    with app.app_context():
+        db.create_all()
+        yield app
+
+    with app.app_context():
+        db.drop_all()
+
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
+
+
+@pytest.fixture
+def two_saved_planets(app):
+    # Arrange
+    planet_mars = Planet(name="Mars",
+                      description="Smallest planet")
+    planet_jupiter = Planet(name="Jupiter",
+                         description="Largest planet")
+
+    db.session.add_all([planet_mars, planet_jupiter])
+
+    db.session.commit()
```

Comment:
I know that the Learn reading has a two_saved_books that looks like this. It creates two objects, adds them and commits to the DB. It would be nice if this fixture (while not shown in Learn) actually returns the objects it creates so that you can use them for comparison in your unit tests, like [this](https://github.com/Ada-C19/flasky/blob/zoisite/tests/conftest.py#L34) and [this](https://github.com/Ada-C19/flasky/blob/zoisite/tests/test_routes.py#L12-L18)